client/src/Service/Audit/AuditEvents.php

```diff
@@ -286,6 +290,50 @@ public function selfRegistrationFailed(array $failureData): array
         return $event + $this->baseEvent(AuditEvents::EVENT_USER_SELF_REGISTER_FAILED);
     }
 
+    /**
+     * @param string $trigger             , what caused the event
+     * @param User   $createdAdminManager , the newly created admin manager
+     * @param User   $currentUser         , the logged in user
+     *
+     * @throws \Exception
+     */
+    public function adminManagerCreated(string $trigger, User $currentUser, User $createdAdminManager): array
+    {
+        $event = [
+            'trigger' => $trigger,
+            'logged_in_user_first_name' => $currentUser->getFirstname(),
+            'logged_in_last_name' => $currentUser->getLastname(),
```

Comment:
It may be useful to include the logged-in users' email here too in case we need to contact or better identify them after looking at the logs.